tests/Api/Admin/StatisticsTest.php

```diff
@@ -150,6 +150,62 @@ public function it_returns_a_not_found_status_code_if_channel_does_not_exist():
         $this->assertResponseCode($this->client->getResponse(), Response::HTTP_NOT_FOUND);
     }
 
+    /** @test */
+    public function it_returns_a_validation_error_if_start_date_is_after_end_date(): void
+    {
+        $this->loadFixturesFromFiles(['authentication/api_administrator.yaml', 'channel.yaml']);
+
+        $this->client->request(
+            method: 'GET',
+            uri: '/api/v2/admin/statistics',
+            parameters: [
+                'channelCode' => 'WEB',
+                'startDate' => '2023-01-01T00:00:00',
+                'interval' => 'month',
+                'endDate' => '2022-01-01T00:00:00',
+            ],
+            server: $this->headerBuilder()->withAdminUserAuthorization('api@example.com')->build(),
+        );
+
+        $this->assertResponseViolations(
+            $this->client->getResponse(),
+            [
+                [
+                    'propertyPath' => '',
+                    'message' => 'The start date must be earlier than the end date.',
+                ],
+            ],
+        );
+    }
```

Comment:
You can merge this with the one below and use a data provider for both cases.
Also, while you're at it, the `::invalidPeriods()` method is unused and can be either adjusted for this test or removed.